portfolio/src/main/java/com/google/sps/ListPuzzles.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps;
+
+import java.util.ArrayList;
+
+public class ListPuzzles {
+
+  private ArrayList<Puzzle> puzzles;
```

Comment:
The type can be loosened a bit here. We don't care too much about whether it's an ArrayList,LinkedList, or any other type of list for that matter so we can just set the type to the List interface.
```suggestion
  private List<Puzzle> puzzles;
```